examples/load/README.md

```diff
@@ -0,0 +1,155 @@
+# File based task/template/handler definition
+
+This proposal introduces directory based task, template, and handler definition.
+
+## Configuration
+
+```
+[load]
+ enabled = true
+ dir="/path/to/directory"
+```
+
+`dir` specifies the directory where the definition files exist.
+
+The specified directory will check to see if any of three subdirectories `tasks`, `templates`, and `handlers` exists.
+
+The `tasks` directory will contain will contain task tickscripts and and the associated templated task
+definition files (either yaml or json).
+
+The `templates` directory will contain both template tickscripts
+
+The `handlers` directory will contain will contain all topic handler definitions in yaml or json.
```

Comment:
The `handlers` directory will contain will contain topic handler definitions in yaml or json.

Reply:
fixed